parachain-tracer/src/prometheus.rs

```diff
@@ -75,6 +75,10 @@ struct MetricsInner {
 	para_block_times_sec: HistogramVec,
 	/// Parachain's on-demand orders
 	para_on_demand_orders: GaugeVec,
+	/// Latency between ordering a slot by a parachain and its last included candidate in relay blocks
+	para_on_demand_delay: GaugeVec,
+	/// Latency between ordering a slot by a parachain and its last included candidate in seconds
```

Comment:
The description is not accurate, the candidate is considered included when it becomes availability, in our case the latency should measure the time to being backed.